helpers/helper-translation-discussion.php

```diff
@@ -519,6 +510,27 @@ public function sanitize_translation_id( $translation_id ) {
 		}
 		return $translation_id;
 	}
+
+	/**
+	 * Kills WordPress execution and displays HTML page with an error message if the original id is incorrect.
+	 *
+	 * Used as callback to validate the original_id passed on rejecting a string with feedback
```

Comment:
```suggestion
	 * Callback to validate the comment meta original_id.
```

Reply:
I'm not sure we want to change this, since we don't save the original_id as a comment meta, we only use it to get the post_id

Reply:
Right. Can we change it so that it `throw`s an exception rather than `wp_die()`?